rest/route/host_test.go

```diff
@@ -863,6 +865,51 @@ func TestClearHostsHandler(t *testing.T) {
 	assert.Equal(t, res.TerminatedVolumes[0], "v1")
 }
 
+func TestRemoveAdminHandler(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(dbModel.ProjectRefCollection))
+	projectRef0 := &dbModel.ProjectRef{
+		Owner:     "mongodb",
+		Repo:      "test_repo0",
+		Branch:    "master",
+		RepoKind:  "github",
+		Enabled:   true,
+		BatchTime: 10,
+		Id:        "test0",
+		Admins:    []string{"user1", "user0"},
+	}
+	projectRef1 := &dbModel.ProjectRef{
+		Owner:     "mongodb",
+		Repo:      "test_repo1",
+		Branch:    "master",
+		RepoKind:  "github",
+		Enabled:   true,
+		BatchTime: 10,
+		Id:        "test1",
+		Admins:    []string{"user1", "user2"},
+	}
+
+	assert.NoError(t, projectRef0.Insert())
+	assert.NoError(t, projectRef1.Insert())
+
+	offboardedUser := "user0"
+	handler := offboardUserHandler{
+		sc:     &data.DBConnector{},
+		dryRun: false,
```

Comment:
It may be beneficial to first test with dryRun = true, and verify that projectRef0 still has user0, and then set dryRun = false.